apm-sniffer/apm-agent/src/main/java/org/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -103,6 +106,8 @@ public void onTransformation(TypeDescription typeDescription, ClassLoader classL
                 if (logger.isDebugEnable()) {
                     logger.debug("On Transformation class {}.", typeDescription.getName());
                 }
+
+                InstrumentDebuggingClass.INSTANCE.log(typeDescription, dynamicType);
```

Comment:
Maybe we can use the `dynamicType.saveIn(File)` API?

Reply:
@ascrutae Good suggestion. Adjustments have been done.